heapster-saw/examples/Dilithium2.saw

```diff
@@ -2,6 +2,18 @@ enable_experimental;
 
 import "Dilithium2.cry";
 
+// The required `dilithium2.bc` file is to be built by:
+// 1. Cloning the `standard` branch of the official Dilithium reference
+//    implementation (https://github.com/pq-crystals/dilithium)
+// 2. Applying the `dilithium.patch` file provided in this directory
+// 3. Running `LLVM_COMPILER=clang make bitcode` in the `ref` directory of the
```

Comment:
Should the `LLVM_COMPILER=clang` bit be part of the `Makefile` target for compiling Dilithium?

Reply:
I'm not sure exactly what you mean by this. Perhaps you're thinking this call to `make` uses the `Makefile` from `heapster-saw/examples`, where the `dilithium` target is? It is not, this is using the `Makefile` of the `dilithium` repo (thus the "in the `ref` directory" bit). Otherwise I'm not sure, because this command [is part of the `Dilithium.bc` target](https://github.com/GaloisInc/saw-script/blob/2ac59d64affacb4620f9ea0108cea5aa15371dd8/heapster-saw/examples/Makefile#L49) in the `heapster-saw/examples` `Makefile`.

Reply:
Silly me, I looked in the `dilithium` target, not the `dilithium%.bc` target (which wasn't visible in the diff). Sorry for the noise!